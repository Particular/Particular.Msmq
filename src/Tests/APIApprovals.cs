namespace Tests
{
    using NUnit.Framework;
    using Particular.Approvals;
    using Particular.Msmq;
    using PublicApiGenerator;

    [TestFixture]
    public class APIApprovals
    {
        [Test]
        public void ApproveApi()
        {
            var publicApi = typeof(MessageQueue).Assembly.GeneratePublicApi(new ApiGeneratorOptions
            {
                ExcludeAttributes = ["System.Runtime.Versioning.TargetFrameworkAttribute", "System.Reflection.AssemblyMetadataAttribute"]
            });

            Approver.Verify(publicApi);
        }
    }
}
