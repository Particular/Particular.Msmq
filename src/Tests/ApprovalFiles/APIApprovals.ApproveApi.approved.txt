[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Tests")]
[assembly: System.Runtime.Versioning.SupportedOSPlatform("Windows7.0")]
[assembly: System.Runtime.Versioning.TargetPlatform("Windows7.0")]
